webapp/sources/api-doc/components/schemas/timezone.yml

```diff
@@ -13,4 +13,4 @@ properties:
   offset:
     type: string
     description: Timezone offset compared to UTC, in +/-HHMM format
-    example: +0200
+    example: "+0200"
```

Comment:
not related to the change, but currently the doc is incorrect, showing int `200` in place of string `+0200`